native/packages/react-native-bpk-component-banner-alert/src/BpkBannerAlert.android.js

```diff
@@ -0,0 +1,241 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  View,
+  StyleSheet,
+  ViewPropTypes,
+  TouchableNativeFeedback,
+  TouchableWithoutFeedback,
+} from 'react-native';
+
+import {
+  colorGray50,
+  colorGray500,
+  colorGreen500,
+  colorBlue500,
+  colorRed500,
+  colorYellow500,
+  spacingBase,
+  spacingSm,
+  spacingLg,
+  spacingXxl,
+} from 'bpk-tokens/tokens/base.react.native';
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import BpkText from 'react-native-bpk-component-text';
+import BpkAnimateHeight from 'react-native-bpk-component-animate-height';
+import BpkIcon from 'react-native-bpk-component-icon';
+
+import { dismissablePropType } from './customPropTypes';
+
+import AnimateAndFade from './AnimateAndFade';
+
+export const ALERT_TYPES = {
+  SUCCESS: 'success',
+  WARN: 'warn',
+  ERROR: 'error',
+  NEUTRAL: 'neutral',
+};
```

Comment:
Could (should?) we abstract `ALERT_TYPES` into a file that iOS and Android can import, to remove the duplication?